tests/standalone-vault/provider.tf

```diff
@@ -0,0 +1,19 @@
+provider "aws" {
+  region = local.region
```

Comment:
We can rely on `AWS_DEFAULT_REGION` being set for the provider once https://github.com/hashicorp/ptfe-replicated/pull/820 is merged.

Reply:
Cool, I'll submit another PR after merging.